jaxws-ri/boms/bom-ext/pom.xml

```diff
@@ -99,6 +100,16 @@
                 <artifactId>eclipselink</artifactId>
```

Comment:
you may want to remove this particular dependency as it includes 'moxy' which you've added. I believe that what is needed are: 'javax.persistence-api', 'org.eclipse.persistence.moxy', 'org.eclipse.persistence.core', 'org.eclipse.persistence.asm' and for sdo 'org.eclipse.persistence.sdo' and 'commonj.sdo'. Note that for sdo there is currently no way to avoid split package problem

Reply:
Updated.